workflow/applications/gfs_cycled.py

```diff
@@ -230,6 +230,10 @@ def get_task_names(self):
 
         if self.do_gempak:
             gfs_tasks += ['gempak']
+            gfs_tasks += ['gempakmeta']
+            gfs_tasks += ['gempakmetancdc']
```

Comment:
If this is meant for only gdas then you need to add this task to the `gdas_tasks` above. If this is only for gfs then it's fine but see other comment. If both then add to `gdas_tasks`.

Reply:
gempakmetancdc should be gdas-only. others are gfs-only.

Reply:
Done.